src/TwigComponent/src/Command/TwigComponentDebugCommand.php

```diff
@@ -314,4 +334,16 @@ private function getAnonymousComponentProperties(ComponentMetadata $metadata): a
 
         return $properties;
     }
+
+    private function resolveEventsListening(string $class): array
```

Comment:
The duplication of this method is unfortunate. Can we use/adjust [this method](https://github.com/symfony/ux/blob/a13900f005c88c6548ebf9daa2ebb79da8340614/src/LiveComponent/src/Attribute/AsLiveComponent.php#L131)?

Reply:
Thanks for the suggestion ! I was not aware of this method. I removed the @internal to use it, but don't you think it could be a separate service ?

Reply:
> I removed the https://github.com/internal to use it

Can you keep `@internal` on the method. We don't want this to be used by end-users.

> but don't you think it could be a separate service ?

Can you explain this a bit more?

Reply:
> Can you explain this a bit more?
Since the method is use elsewhere and its logic is not inherently linked to the attribute, I would think of creating a separate service let's say `LiveListenersResolver` and call it in the attribute and in my modifications.

Reply:
Fair point, keeping it internal enables us to make this kind of refactor in the future. For now, let's keep as is.